cirq/google/engine/engine.py

```diff
@@ -152,47 +156,43 @@ class Engine:
     """
 
     def __init__(self,
-                 api_key: Optional[str] = None,
-                 api: str = 'quantum',
+                 project_id: str,
                  version: str = 'v1alpha1',
-                 default_project_id: Optional[str] = None,
                  discovery_url: Optional[str] = None,
                  default_gcs_prefix: Optional[str] = None,
                  proto_version: ProtoVersion = ProtoVersion.V1,
                  **kwargs) -> None:
         """Engine service client.
 
+        Requires project_id.
```

Comment:
Don't think you have to say this since any field without a default value will yell in python

Reply:
Done